Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -394,13 +392,16 @@
     private static int LATCH_WAIT_UNTIL = 2500; //used for getLatestReferringParamsSync and getFirstReferringParamsSync, fail after this many milliseconds
 
     /* List of keys whose values are collected from the Intent Extra.*/
-    private static final String[] EXTERNAL_INTENT_EXTRA_KEY_WHITE_LIST = new String[] {
+    private static final String[] EXTERNAL_INTENT_EXTRA_KEY_WHITE_LIST = new String[]{
             "extra_launch_uri"   // Key for embedded uri in FB ads triggered intents
     };
 
     private CountDownLatch getFirstReferringParamsLatch = null;
     private CountDownLatch getLatestReferringParamsLatch = null;
 
+    /* Flag for checking of Strong matching is waiting on GAID fetch */
+    private boolean performCookieBaseDStrongMatchingOnGAIDAvailable = false;
```

Comment:
performCookieBaseDStrongMatchingOnGAIDAvailable - is the "D" upper case for a reason or typo?